docs/ips.md

```diff
@@ -0,0 +1,37 @@
+# IP Allocation
+
+## Production
+
+| Application/Node          | Type               |             IP/CIDR             |
+| ------------------------- | ------------------ | :-----------------------------: |
+| keepalived                | VIP                |          192.168.1.200          |
+| k8s-controlplane-01       | Control-Plane Node |          192.168.1.121          |
+| k8s-controlplane-02       | Control-Plane Node |          192.168.1.122          |
+| k8s-controlplane-03       | Control-Plane Node |          192.168.1.123          |
+| k8s-node-01               | Node               |          192.168.1.131          |
+| k8s-node-01               | Node               |          192.168.1.132          |
+| k8s-node-01               | Node               |          192.168.1.133          |
+| metallb                   | Daemonset          | 192.168.1.150 <-> 192.168.1.155 |
+| istio                     | LoadBalancer       |          192.168.1.150          |
+| coredns                   | LoadBalancer       |          192.168.1.151          |
+| mosquitto                 | LoadBalancer       |          192.168.1.152          |
+| zigbee2mqtt               | LoadBalancer       |          192.168.1.153          |
+| zigbee2mqtt (code-server) | LoadBalancer       |          192.168.1.154          |
```

Comment:
One proposal I have for this is if it's a static IP use a /32 for example

| Application/Node          | Type               |             IP/CIDR             |
| ------------------------- | ------------------ | :-----------------------------: |
| zigbee2mqtt (code-server) | LoadBalancer       |          192.168.1.154 /32         |

Reply:
Great idea, I'll make the adjustment.